src/scripts/main.js

```diff
@@ -18,10 +18,26 @@ const food = {
   },
 };
 
-const tree = document.querySelector('#tree');
+let tree = document.querySelector('#tree');
+
+tree = document.createElement('div');
+tree.id = 'tree';
+document.body.appendChild(tree);
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(li, data[key]);
+    }
+    ul.appendChild(li);
+  }
+  element.appendChild(ul);
 }
 
 createTree(tree, food);
```

Comment:
Here you are calling the `createTree` function with `tree` and `food` as arguments. However, the `tree` variable is a div element, not an ul or li element, which means the created list will be a child of this div, not a child of a list element. Consider creating an initial ul element to serve as the root of the tree.